### PR TITLE
Add collabora online

### PR DESCRIPTION
**Pull request by LiquidITGuy**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [v] I have tested the template using the method described in README.md thoroughly
- [v] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [v] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [v] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [v] Icon is added as a png file to the logos directory.
- [v] Documentation field contains a link to a page with proper explanations on environmental variables, volumes or the docker compose file.

### CODE REVIEW

**Comment by LiquidITGuy** [comment]
Be careful, I wrote this code fully inspired (and half the code is paste from) https://github.com/caprover/one-click-apps/pull/111

I update it, try it and rewrote some description but it's more than just little bit inspired (but I do not know how to wrote it)
This version is the one I use on my Nextcloud server

**Comment by LiquidITGuy** [comment]
Capture d’écran 2020-06-22 à 23 22 23
Capture d’écran 2020-06-22 à 23 22 56

File: public/v2/apps/collabora-online.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      "label": "Collabora CODE Version",
+      "defaultValue": "4.2.4.4",
+      "description": "Checkout their docker page for the valid tags https://hub.docker.com/r/collabora/code/tags/",
+      "validRegex": "/^([^\\s^\\/])+$/"
+    },
+    {
+      "id": "$$cap_wopihost_domain",
+      "label": "WOPI host domain",
+      "defaultValue": "example\\.domain\\.tld",
+      "TODO" : "rename default value with better example",
```
Comment (githubsaturn) [inline public/v2/apps/collabora-online.json:52]: Change the default to `$$cap_appname.$$cap_root_domain` ?

Comment (LiquidITGuy) [reply public/v2/apps/collabora-online.json]: I changed the description and thz default value (this is the nextcloud url and not the url of the code instance)

**Pull request merged**